src/app/api-server/Cargo.toml

```diff
@@ -69,7 +69,7 @@ kamu-auth-rebac-sqlite = { workspace = true }
 kamu-datasets = { workspace = true }
 kamu-datasets-inmem = { workspace = true }
 kamu-datasets-postgres = { workspace = true }
-kamu-datasets-services = { workspace = true }
+kamu-datasets-services = { workspace = true, default-features = false, features = ["lfs", "s3"] }
```

Comment:
Isn't workspace already enabling those features?

Reply:
I've moved the enabling of features inside the app